app/scripts/modules/tencentcloud/src/pipeline/stages/shrinkCluster/shrinkClusterStage.ts

```diff
@@ -0,0 +1,11 @@
+import { validate } from './shrinkClusterValidators';
+import { Registry, IStageTypeConfig } from '@spinnaker/core';
```

Comment:
```suggestion
import { Registry } from '@spinnaker/core';
```
I think now CI is failing because of this unused import.

Reply:
yup, removed